[2023-10-20 23:22:13,739] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2023-03-18T00:00:00+00:00 [queued]>
[2023-10-20 23:22:13,757] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2023-03-18T00:00:00+00:00 [queued]>
[2023-10-20 23:22:13,757] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-10-20 23:22:13,759] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2023-10-20 23:22:13,760] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-10-20 23:22:13,783] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): segunda_tarea> on 2023-03-18 00:00:00+00:00
[2023-10-20 23:22:13,788] {standard_task_runner.py:52} INFO - Started process 2334 to run task
[2023-10-20 23:22:13,792] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'mi_primer_dag_v3', 'segunda_tarea', 'scheduled__2023-03-18T00:00:00+00:00', '--job-id', '693', '--raw', '--subdir', 'DAGS_FOLDER/primer_dag_v3.py', '--cfg-path', '/tmp/tmpo4japq0d', '--error-file', '/tmp/tmp8f9_pe1_']
[2023-10-20 23:22:13,793] {standard_task_runner.py:80} INFO - Job 693: Subtask segunda_tarea
[2023-10-20 23:22:13,991] {task_command.py:371} INFO - Running <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2023-03-18T00:00:00+00:00 [running]> on host dfdda228c9fc
[2023-10-20 23:22:14,075] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=DavidBU
AIRFLOW_CTX_DAG_ID=mi_primer_dag_v3
AIRFLOW_CTX_TASK_ID=segunda_tarea
AIRFLOW_CTX_EXECUTION_DATE=2023-03-18T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-03-18T00:00:00+00:00
[2023-10-20 23:22:14,076] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-10-20 23:22:14,077] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hola, soy la tarea 2 y sere corrida luego de la Tarea 1']
[2023-10-20 23:22:14,087] {subprocess.py:85} INFO - Output:
[2023-10-20 23:22:14,088] {subprocess.py:92} INFO - hola, soy la tarea 2 y sere corrida luego de la Tarea 1
[2023-10-20 23:22:14,089] {subprocess.py:96} INFO - Command exited with return code 0
[2023-10-20 23:22:14,118] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=mi_primer_dag_v3, task_id=segunda_tarea, execution_date=20230318T000000, start_date=20231020T232213, end_date=20231020T232214
[2023-10-20 23:22:14,165] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-10-20 23:22:14,203] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
